docs-chef-io/content/inspec/resources/timezone.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "timezone resource"
+draft = false
+gh_repo = "inspec"
+platform = "linux"
+
+[menu]
+  [menu.inspec]
+    title = "timezone"
+    identifier = "inspec/resources/os/timezone.md timezone resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `timezone` Chef InSpec audit resource to test timezone configurations of the system.
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
+
+A `timezone` resource fetches the timezone configurations of the system and compares the output with the test:
+
+    describe timezone do
+      its('property') { should eq 'expected value' }
+    end
+
+where
+
+- `'property'` is one of `identifier` , `name` and `time_offset`
+- `'expected value'` tests the output of the command run on the system versus the expected output stated in the test
+
+For example:
+
+    describe timezone do
+      its('identifier') { should eq 'Asia/Kolkata' }
+      its('name') { should eq 'IST' }
+      its('time_offset') { should eq '+0530' }
+    end
+
+## Properties
+
+### identifier
+
+The `identifier` property returns the location identifier of the timezone.
+
+An example of checking the **identifier** for India Time :
+
+    its('identifier') { should eq 'Asia/Kolkata }
+
+### name
+
+The `name` property returns the name of the timezone.
+
+An example of checking the **name** of India Time :
```

Comment:
I probably read more about time zones for this PR than I should have, but it turns out that these abbreviations aren't standardized and so there are [several ISTs and several CSTs](https://en.wikipedia.org/wiki/List_of_time_zone_abbreviations). Anyway, it looks like this test would pass if a system was set in Ireland, Israel, or India, and the only way to resolve which time zone the system has is to use one of the other properties.

```suggestion
An example of checking the **name** for the IST time zone abbreviation:
```